packages/web-react/src/components/Box/stories/Box.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import { Container } from '../../Container';
+import Box from '../Box';
+import ReadMe from '../README.md';
+
+const meta: Meta<typeof Box> = {
+  title: 'Components/Box',
+  component: Box,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+  },
+  argTypes: {
+    elementType: {
+      control: 'text',
+    },
+  },
+  args: {
+    elementType: 'div',
+  },
```

Comment:
should't be here more props to set? `background`, `border`, `paddings`

Reply:
Added